### PR TITLE
Correctly format example

### PR DESCRIPTION
**Pull request by Bouke**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/performance/rate-limit.md](https://github.com/dotnet/AspNetCore.Docs/blob/c653d01bdc5c368ba0712cf8f35258ad63246130/aspnetcore/performance/rate-limit.md) | [Rate limiting middleware in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/performance/rate-limit?branch=pr-en-us-30126) |

### CODE REVIEW

**Review by guardrex — APPROVED** [review]
Thanks @Bouke ... and I *grammarized*&trade; it a bit and formatted the table a bit better.

**Comment by guardrex** [comment]
There's another problem here to sort out. The text says ...

> The **Remaining** column ...

... but there's no column with that name.

**Comment by guardrex** [comment]
Looks like it should be the **Available** column.

**Comment by guardrex** [comment]
... and sentences before tables don't get colons ... and I put a little more 🌷 *table gusto* on the next table.

@Bouke ... Do you want to take another look at the final PR and check me? I'm only on my first cup of ☕, so I'm not fully awake yet 😆.

**Comment by guardrex** [comment]
@Rick-Anderson ... You should look this over. I found a few side NITs to touch.

Also in passing, it seems to me that "sliding window" ***as an adjective*** is a compound adjective, so it seems like it should be hyphenated. For example ...

> Consider the following table that shows a sliding-window limiter with a 30-second window, three segments per window, and a limit of 100 requests:

Let me know if you agree and want that change. If so, I'll commit it before merging (along with any other changes you suggest).

File: aspnetcore/performance/rate-limit.md [hunk 1]
```diff
@@ -60,10 +60,12 @@ A sliding window algorithm:
...
 * Each time window is divided in `n` segments per window.
-* Requests taken from the expired time segment one window back (`n` segments prior to the current segment), are added to the current segment. We refer to the most expired time segment one window back as the expired segment.  Consider the following table which shows a sliding window limiter with a 30-second window, 3 segments per window and a limit of 100 requests:
+* Requests taken from the expired time segment one window back (`n` segments prior to the current segment) are added to the current segment. We refer to the most expired time segment one window back as the expired segment.
+
+Consider the following table that shows a sliding window limiter with a 30-second window, three segments per window, and a limit of 100 requests:
 
 * The top row and first column shows the time segment.
-* The second row shows the remaining requests available. The remaining requests are available-requests+recycled.
+* The second row shows the remaining requests available. The remaining requests are calculated as the available requests less the processed requests plus the recycled requests.
```
Comment (Rick-Anderson) [inline aspnetcore/performance/rate-limit.md:68]:
```suggestion
* The second row shows the remaining requests available. The remaining requests are calculated as the available requests minus the processed requests plus the recycled requests.
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@Bouke  you PR will be live in 15 minutes if you want to take a look
Thanks for the PR, nice improvements

**Comment by Bouke** [comment]
> @Bouke ... Do you want to take another look at the final PR and check me? I'm only on my first cup of ☕, so I'm not fully awake yet 😆.

Seeing that It's already reviewed and merged, I'll skip an actual review. I think the changes are looking good.

**Pull request merged**